src/composable/types/Twap.ts

```diff
@@ -101,49 +90,91 @@ export type TwapData = {
    * whether the TWAP is valid for the entire interval or not
    */
   readonly span: BigNumber
+}
 
+/**
+ * Parameters for a TWAP order, made a little more user-friendly for SDK users.
+ *
+ * @see {@link TwapStruct} for the native struct.
+ */
+export interface TwapData extends TwapDataBase {
```

Comment:
define the user friendly data for TWAP from the base. 

It will be more declarative regarding the start time (so you can explicitly ask to start at mining time), and the span (so you can explicitly ask to use the same as the time between parts)

Reply:
Awesome! Very usable.